packages/fluentui/react-bindings/src/styles/types.ts

```diff
@@ -73,28 +66,26 @@ export interface StylesContextPerformance {
 
 export type StylesContextPerformanceInput = Partial<StylesContextPerformance>;
 
-export type StylesContextInputValue<R = Renderer> = {
+export type StylesContextInputValue = {
   rtl?: boolean;
   disableAnimations?: boolean;
   performance?: StylesContextPerformanceInput;
-  renderer?: R;
+  renderer?: Renderer;
   theme?: ThemeInput;
 };
 
-export type StylesContextValue<R = Renderer> = {
+export type StylesContextValue = {
   rtl: boolean;
   disableAnimations: boolean;
   performance: StylesContextPerformance;
-  renderer: R;
+  renderer: Renderer;
```

Comment:
This means that if we want to support some other CSSinJS solution, we should add a wrapper render and that should be it right?

Reply:
You're right/ We will to:
- implement a wrapper renderer that implements `Renderer`
- pass it via `RendererContext`